frontend/kesaseteli/employer/src/hooks/application/useAccordionStateLocalStorage.ts

```diff
@@ -0,0 +1,28 @@
+import { useFormContext } from 'react-hook-form';
```

Comment:
Remembers which accordion was closed and which one is open
<img width="1632" alt="Screenshot 2021-09-20 at 4 02 36 PM" src="https://user-images.githubusercontent.com/7648194/134006810-ae4d673d-72b0-4175-b79a-9fa6a83d57b7.png">